Automattic/wp-calypso
# Update Pattern page header image

## Proposed Changes

* Update the header background image to a PNG to avoid issues with the pattern tiles appearing pixelated.
* Fix alignment so that the right side of the image aligns with the columns below.

## Testing Instructions



* Run the app.
* Visit http://calypso.localhost:3000/patterns.
* Observe that the background image no longer has white borders or is pixelated.

### Before
<img width="1619" alt="Screenshot 2024-03-05 at 13 41 30" src="https://github.com/Automattic/wp-calypso/assets/417538/a1900d0b-0c4a-4e83-83ea-d1923a7a4f58">

### After
<img width="1618" alt="Screenshot 2024-03-05 at 13 41 04" src="https://github.com/Automattic/wp-calypso/assets/417538/1296259b-b6ee-42e5-9111-5a772898ad5e">

## Pre-merge Checklist
